### DISCUSSION TITLE
🎉 Special Thanks, Upcoming Updates & Content Inquiry 🎉

### DISCUSSION THREAD

**Discussion by cromewar** [Cyfrin/foundry-full-course-cu#916] [Announcements]
Hey Web3Edu Community!

First and foremost, a huge shoutout to everyone who's signed up for early access! Your enthusiasm and support mean the world to us. Stay tuned, as we'll be sharing some exciting updates on that front very soon, including videos discussing updated packages and addressing the most common problems faced by our learners. 🚀

Now, as we continue to grow and evolve, we're always looking to provide content that truly resonates with and benefits all of you. With that in mind, we want to hear directly from YOU!

🔍 What additional content would you like to see next?

Whether it's a deep dive into a specific Web3 topic, additional tutorials, or any other area of interest, we're all ears. Your feedback will shape the future of our content, ensuring we cater directly to your needs and interests.

Please comment below with your thoughts, suggestions, or requests. We truly value your input, and we're eager to craft our upcoming materials based on the collective wisdom of this remarkable community.

Together, let's define the next chapters of Web3Education! 🛠️💡

Warm regards,
Vasiliy, Developer Advocate at Cyfrin.

**Comment by theirrationalone** [comment]
1. WEB3_Security.
2. How to Earn 💵💵💲
3. How(Strategies, Advocates Advice) to Build Charismatic WEB3 Resume, Portfolio or anything IYO we should have??
4. How to Play into the Wild(Contest). What mindset we should prepare right before participating into Contests.
5. How to Build our own blockchain, L1s, L2s.
6. What is HLedger????
7. Avid to see at least 3 hacking tricks (Reentrancy Attack included).
8. What the heck BOTS are doing??? How to ab*se em??? (Serious Sorry for that explicit word).
9. Assembly?? Huff???
10. Books we should have as our last guide???

Sorry if i flooded too much.

;`)

**Comment by Jamster187** [comment]
I don't see many junior dev roles for Solidity currently, would love to see a guide where the sole purpose is to help you land a job.

**Comment by shikhar229169** [comment]
To have a clear cut guide that explains how to land a internship/job in web3.
As we see that most of the jobs actually require 2 years of experience or some require more than that, so how can a fresher like us after taking this course can actually get started working for the industry.

**Comment by Ethanol48** [comment]
I would love to see some things:

- In-line Assembly / Yul:  to deepen our understading on the EVM, and know more about bit operations, more low level programming, learn the REAL fundamentals.


And some points I agree with @theirrationalone:

- Books (if available, it would be a great tool)

- ## A resume: 
  - I feel that new developers struggle a bit on these as there isn't a lot of examples of resumes for web3, if you could give on how to create a Resume, how to present ourselves in the industry it would be awesome.

**Comment by shrutic11** [comment]
1. Writing/Designing/ better tests (using foundry) for smart contracts which are gas efficient and also fulfil the required coverage.
2. Auditing an existing smart contract live (as to understand the approach)
3. A couple of projects with Hyperledger.

**Reply by cromewar** [reply]
Thanks for the request ideas

**Comment by sunnyStefi** [comment]
I would focus mainly on **Security**, being backed by the old, stable **craftmaship** 🔨 concept: 

1) 😶‍🌫️ 😶‍🌫️ 😶‍🌫️ education today is always in the realm of 'ideas' : it tries to explain ALL the web3 things with at the end a little practice for each theme

-  ✅ It's true that it's worth to have a general knowledge of web3 (like it's done in Patrick's tutorial) because we need to have a **general** **network** 🕸️ **of knowledge** to build upon
- 🦸🏼‍♂️ 💡 we can also agree that you can make Superheros only if you are a Superhero on that thing first 🦸🏼‍♂️. Since Cyfrin is **Security oriented**, I would focus mainly on teaching that very well. In this way, **everyone** grows for real 🔼

2) 🏭 **learning by crafting in a real industry**: a really honest learning (expecially in a fast pace industry like web3) happens when the student is inserted inside a community of people working towards a **common goal** ⚽ (e.g. enhancing smart contract security everywhere)

3) rewards (money, job, ..) are also very important when someone is learning web3: people that are just interested for its own sake are not that many. Usually the people who are here, want to learn web3 for landing a good/better job to improve their lives 

Keep up the good work! 🏋🏼‍♂️🏋🏼‍♂️🏋🏼‍♂️

**Reply by cromewar** [reply]
Thanks for this cool ideas, the security curriculum is coming soon! 😉

**Comment by Ibby-debug** [comment]
I wanna know who funded the fundFundme fundMe contract
